ExploreASL_Initialize.m

```diff
@@ -1,16 +1,10 @@
-function [x] = ExploreASL_Initialize(DataParPath, ProcessData, iWorker, nWorkers)
+function [x] = ExploreASL_Initialize(varargin)
 %ExploreASL_Initialize Initializes ExploreASL
 %
-% FORMAT: [x] = ExploreASL_Initialize([DataParPath, ProcessData, iWorker, nWorkers])
+% FORMAT: [x] = ExploreASL_Initialize([DataParPath, ImportArray, ProcessArray, SkipPause, iWorker, nWorkers])
 %
 % INPUT:
-%   DataParPath - path to data parameter file (OPTIONAL, required when ProcessData=true, will then be prompted if omitted)
-%   ProcessData - 0 = only initialize ExploreASL functionality (e.g. path management etc, REQUIRED, will be prompted if omitted)
-%               - 1 = initialize ExploreASL functionality, load data & start processing pipeline, 
-%               - 2 = initialize ExploreASL functionality, load data but no processing
-%               - (OPTIONAL, default=prompt the user)
-%   iWorker     - allows parallelization when called externally. iWorker defines which of the parallel ExploreASL calls we are (OPTIONAL, DEFAULT=1)
-%   nWorkers    - allows parallelization when called externally. nWorkers defines how many ExploreASL calls are made in parallel (OPTIONAL, DEFAULT=1)
+%   This script can accept the same arguments as ExploreASL_Master. Check out the definitions there.
```

Comment:
Good point!

Reply:
BTW: the guide for good practice in comments and commit messages says "can accept" -> "accepts"

Reply:
Fixed 2cf728a.